### ISSUE TITLE
urlQuotes throwing warning for LESS variable

### ISSUE THREAD

**Issue by amandakalk** [lesshint/lesshint#364]
**Which version of `lesshint` are you using?**
3.1.0

**How are you running `lesshint`? CLI, Node.js API, Grunt/Gulp plugin?**
Gulp

**What's your `.lesshintrc` configuration?**

```js
{
    "fileExtensions": [".less"],
    "spaceAfterPropertyColon": {
        "enabled": true,
        "style": "one_space" // Comments are allowed
    },
    "propertyOrdering": {
        "enabled": false
    },
    "maxCharPerLine": {
        "enabled": false
    }
}
```

**If you're reporting a bug, please show us some code that's failing.**

```less
@brand-logo: '/assets/images/logo.svg';

.head-logo {
	background: url(@brand-logo);
}
```

**What did you expect to happen?**
No warning. The variable evaluates to a properly quoted url.

**What's the actual result?**
I get this warning: `urlQuotes: URLs should be enclosed in quotes`

Technically, I could write the background declaration as `background: url('@{brand-logo}');` to prevent the warning, but it feels weird to add unnecessary interpolation just to make the linter happy. Thoughts?

**Comment by jwilsson** [comment]
Hi,
Thanks for the report!

I think it would make sense for it to ignore variables. If the rest of the team agrees, would you be interested in submitting a PR @amandakalk?

**Comment by kokarn** [comment]
👍 

Would this come up somewhere else or maybe break in some situations? 
I'm looking at the [documentation](http://lesscss.org/features/#features-overview-feature-variables) and if I'm not mistaken you could end up doing something like

```less
@randomstring: aString;

#header {
  background-image: url( @randomstring );
}
```

Does this matter in any case or can we safely ignore all the vars?

**Comment by amandakalk** [comment]
If you ignore variables completely, this would compile and not throw the error:
```less
@randomstring: test.png;

#header {
  background-image: url( @randomstring );
}
```
This **would not** compile at all (unquoted string with slash):
```less
@randomstring: assets/test.png;

#header {
  background-image: url( @randomstring );
}
```

So technically you could be ignoring a failure, but seems kind of unlikely that your URL wouldn't contain a slash in it.

**Comment by jwilsson** [comment]
Thanks for the extra examples!

I agree that a URL without a slash seems unlikely, and we can never be sure of what the variable actually contains. I still think just ignoring variables in `url()` is fine (FWIW I checked other linters and they do it too).

A PR would be highly appreciated! Let us know if you need anything.

**Comment by jwilsson** [comment]
Closed in #366.
